test/resources/language/erlang-rebar3/rebar.config

```diff
@@ -0,0 +1,5 @@
+{erl_opts, 
+	[
+		return_warnings
+	]}.
+{deps, []}.
```

Comment:
Yes! The checker determines its ability to run based on the presence of this file.